client/app/lib/providers/computecontroller.coffee

```diff
@@ -997,6 +999,47 @@ module.exports = class ComputeController extends KDController
           @emit 'StacksInconsistent', stack
 
 
+  sharingStackTemplate: (_id, type) ->
+    remote.api.JStackTemplate.one { _id }, (err, template) ->
+      template.setAccess type
```

Comment:
do you want me to update this function name?
https://github.com/koding/koding/blob/master/workers/social/lib/social/models/computeproviders/stacktemplate.coffee#L56